### ISSUE TITLE
Pain points with Joplin styling

### ISSUE THREAD

**Issue by Niicck** [cityofaustin/techstack#4530]
Recently, I had to add new templates for publish notifications. This is currently complex and widely spread out. There are multiple places where things must be edited:

- I wanted to create new templates. So I added them under `templates/Joplin_UI/messages` and `templates/Joplin_UI/pageStatus`. This made sense to me.
- They had corresponding js requirements, so I added them to `js/pageStatus/index.js`. The `js/` directory could probably be folded into Joplin_UI.
- Each .js file requires its own webpack loader at this point, and they must be individually added to `webpack.base.js`. And that new bundle must be added into the base template `templates/wagtailadmin/base.html`. We'd probably be fine just loading and running one bundle, like this company: https://tech.gadventures.com/integrating-react-into-django-5e6591097b9e

**Comment by Niicck** [comment]
Convo from wagtail support that may or may not be useful:
> mroobert  9:03 AM
Hey guys, do you know how to override a js file from wagtail/admin app? In detail, I want to override the js files from wagtail/admin/static_src/wagtailadmin/js/blocks. I was thinking to import new js files with @hooks.register('insert_editor_js', order=1000) to import them after  the existing ones in the "view page source" but no matter what order value I put there it doesn't overlap the existing ones... any thoughts? THX :+1:






mattwestcott:point_right:  9:20 AM
@mroobert That would only work if the JS files within Wagtail were also being inserted through the insert_editor_js hook, and usually streamfield JS is inserted through form media instead.
9:21
If you really wanted to override the existing file, you could add a new app above wagtail.admin in INSTALLED_APPS, and place a file within that app at the matching path under static (i.e. static/wagtailadmin/js/blocks )...

**Comment by easherma** [comment]
Yeah, inserting css or js thru the hooks could be a way to simplify your organization/build process for admin stuff: 
https://learnwagtail.com/tutorials/adding-custom-css-and-javascript-to-your-wagtail-admin-area/
